tox.ini

```diff
@@ -6,9 +6,9 @@ passenv = TEMPEST_CONFIG_DIR
 setenv =
   OS_TEST_PATH={toxinidir}/f5lbaasdriver/test/tempest/tests
 deps =
-  -rrequirements.test.txt
+  pytest
 changedir = f5lbaasdriver/test/tempest/tests/
-commands = py.test -vra {posargs}
+commands = py.test {posargs}
```

Comment:
Will dropping the '-vra' args cause any problems in trtl parsing results, or will we lose verbosity since we're removing -v?